pom.xml

```diff
@@ -23,7 +23,8 @@
   </licenses>
   <properties>
     <java.level>7</java.level>
-    <jenkins.version>1.625.3</jenkins.version>
+    <jenkins.version>1.642.3</jenkins.version>
+    <workflow.version>2.6</workflow.version>
```

Comment:
This property does not make much sense, because Pipeline plugin releases are not synchronized starting from 2.0 versions

Reply:
can be change is a following PR, I'll merge in the current state. Thank you for reviewing this @oleg-nenashev.